python/res/enkf/analysis_config.py

```diff
@@ -56,18 +57,26 @@ class AnalysisConfig(BaseCClass):
     _get_global_std_scaling = ResPrototype("double analysis_config_get_global_std_scaling(analysis_config)")
 
 
-    def __init__(self, user_config_file=None):
-        if user_config_file is not None and not isfile(user_config_file):
-            raise IOError('No such configuration file "%s".' % user_config_file)
+    def __init__(self, user_config_file = None, config_content = None):
```

Comment:
I guess we expect exactly one of these to be not `None`? Is so, I suggest to raise an `Exception` if this is not true...

Reply:
Fixed